pkg/convert/tf.go

```diff
@@ -993,6 +993,10 @@ func convertObjectConsExpr(state *convertState, inBlock bool, scopes *scopes,
 			Value: valueTokens,
 		})
 	}
+	// Sort the items by name to make the output deterministic
```

Comment:
Try and sort by HCL source position. That keeps things in the declaration order from the original.

Reply:
Okay that makes more sense

Reply:
Done ✅